### PR TITLE
fixes for Varays master #1080

### PR DESCRIPTION
**Pull request by Vayras**
made the changes

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Thanks Vayras. Can you post a screenshot? This also addresses #1015

**Comment by Vayras** [comment]
Hi, I assume you are asking about the screenshot for the Bounties Superadmin  here you go
Result1
Result2
For now I have center the main container so that the UI does not stretch much on widescreens

**Comment by ecurrencyhodler** [comment]
LGTM. Let's get one more code review.

**Comment by ecurrencyhodler** [comment]
Actually you have a couple of failing tests too. Can you address those?

**Comment by Vayras** [comment]
> Actually you have a couple of failing tests too. Can you address those?

Will do right away

**Comment by Vayras** [comment]
@ecurrencyhodler  just ran prettier check  and added the correct import , I think it's good to go

**Comment by kevkevinpal** [comment]
closed the PR I had for your branch just linking the closed PR so I can see any of the comments quickly
https://github.com/stakwork/sphinx-tribes/pull/1051

**Comment by Vayras** [comment]
So the test's are still failing apparently , even though I have checked the files and formatting

**Comment by ecurrencyhodler** [comment]
Do you have any insight on the failed tests @kevkevinpal ?

Also @Vayras I met with the team. Looks like we won't need issue #1003.

But we noticed that it was included in this PR so we'll pay you out for it still. Woudl you mind removing the initial log in page for #1003 and have community.sphinx.chat/admin go straight to the superadmin frontend in #1015?

**Comment by Vayras** [comment]
@ecurrencyhodler , Thanks for letting me know I will change the route and push the commit , Do you also want me to delete the folder Auth That has files related to #1003 ?

**Comment by ecurrencyhodler** [comment]
@kevkevinpal wdyt?

> "Do you also want me to delete the folder Auth That has files related to #1003?

**Comment by kevkevinpal** [comment]
> @kevkevinpal wdyt?
> 
> > "Do you also want me to delete the folder Auth That has files related to #1003?

ya delete it if it is not being used we should not keep dead code

**Comment by kevkevinpal** [comment]
@Vayras looks like build is failing
```
Module not found: Error: Can't resolve './HeaderStyles' in '/home/runner/work/sphinx-tribes/sphinx-tribes/frontend/app/src/pages/superadmin/Header'
```

you can reproduce if you do `cd ./frontend/src && yarn run build`

**Comment by Vayras** [comment]
@kevkevinpal  I have changed the file name , But I don't think it is updating via the commit , can you please check this?
capitalfilename

**Comment by Vayras** [comment]
@ecurrencyhodler  @kevkevinpal  So, the test's are still failing , I have ran the prettier check on my codebase and  there are no errors also I have capitalized the filename as well , did not update in the commit somehow ,let me know if there is something else I can do to fix this ,Thanks

File: frontend/app/src/config/host.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export function getHost(): string {
 
 export function getHostIncludingDockerHosts() {
   if (externalDockerHosts.includes(window.location.host)) {
-    return 'tribes.sphinx:5002';
+    return 'tribes.sphinx.5002';
```
Comment (kevkevinpal) [inline frontend/app/src/config/host.ts:11]: lets revert this change

File: package.json
Comment (kevkevinpal) [inline package.json:1]: we dont need this package.json at the top level

File: sphinx-tribes.exe
Comment (kevkevinpal) [inline sphinx-tribes.exe:1]: we can remove the sphinx-tribes.exe

**Comment by Vayras** [comment]
@kevkevinpal  Thanks for helping me out with this.

**Comment by ecurrencyhodler** [comment]
I tested on staging and it works! I'm going to have our designer take a look at the container fix you used. I might have a follow up bounty for you vayras. I'll keep you posted. Paying you out now.

**Comment by Vayras** [comment]
> I tested on staging and it works! I'm going to have our designer take a look at the container fix you used. I might have a follow up bounty for you vayras. I'll keep you posted. Paying you out now.

Of course, Keep me posted looking forward to contribute on this

**Comment by ecurrencyhodler** [comment]
Okay Vayras. Designer said let's try to do full width and see how it looks. Is there anyway you can do that locally and post a screenshot of what it would look like?

**Comment by Vayras** [comment]
Let me check , I guess it won't break but it will look just a bit too stretched, Iet me check and post a screenshot for this

**Comment by Vayras** [comment]
@ecurrencyhodler This is how it will look when not assigned fixed width 
Screenshot 2023-12-19 082556

**Comment by ecurrencyhodler** [comment]
@vayras our designer said let's do it without a fixed width. Can you submit this pr?

**Comment by Vayras** [comment]
@ecurrencyhodler  yeah sure , just a minute

**Comment by Vayras** [comment]
#1163 here is the PR
